src/style.css

```diff
@@ -1 +1,109 @@
-/* add styles here */
+html {
+  font-weight: 300;
+}
+
+:root {
+  color: #3D4E61;
```

Comment:
Move all repeatable colors to the css variables.

Reply:
Check all other similar cases.

Reply:
[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)